src/components/TheUpdateDialog.vue

```diff
@@ -17,7 +17,7 @@
 
 <template>
     <v-dialog :value="application !== ''" persistent max-width="800" class="mx-0">
-        <v-card dark :loading="!complete">
+        <v-card :loading="!complete">
```

Comment:
I think this will be wrong, if you just delete the dark option with no condition.